torch/csrc/jit/api/object.h

```diff
@@ -2,11 +2,18 @@
 
 #include <ATen/core/functional.h>
 #include <ATen/core/ivalue.h>
+#include <c10/util/Optional.h>
 #include <torch/csrc/jit/api/method.h>
 
 namespace torch {
 namespace jit {
 
+struct ObjectProperty {
+  std::string name;
+  Method getter_func;
+  c10::optional<Method> setter_func;
+};
```

Comment:
Consider putting this definition inside `Object`; it's not really useful as a class in itself.

Reply:
sure